tests/robustness/multiclient_test/framework/snapshotter.go

```diff
@@ -67,6 +74,20 @@ func (mcs *MultiClientSnapshotter) ConnectOrCreateRepo(repoPath string) error {
 	return err
 }
 
+// SetCacheSizeLimits sets the cache size limits for an existing repository
+// the repository server is connected to.
+// Allowed flags: content-cache-size-limit-mb, metadata-cache-size-limit-mb.
+func (mcs *MultiClientSnapshotter) SetCacheSizeLimits(cacheFlag string, cacheSize int) error {
```

Comment:
does this need to be exported? it seems to only be consumed by `package framework`?

Reply:
you are correct, updated.